tests/types/resolving/test_unions.py

```diff
@@ -44,49 +40,18 @@ class User:
     class Error:
         name: str
 
-    @strawberry.type
-    class Query:
-        user: Optional[Union[User, Error]]
-
-    definition = Query._type_definition
-
-    assert definition.name == "Query"
-    assert len(definition.fields) == 1
+    cool_union = union(name="CoolUnion", types=(User, Error))
+    annotation = StrawberryAnnotation(cool_union)
+    resolved = annotation.resolve()
 
-    assert definition.fields[0].graphql_name == "user"
-    assert isinstance(definition.fields[0].type, StrawberryOptional)
+    assert isinstance(resolved, StrawberryUnion)
+    assert resolved.types == (User, Error)
 
-    strawberry_union = definition.fields[0].type.of_type
-    assert isinstance(strawberry_union, StrawberryUnion)
-    assert strawberry_union.name == "UserError"
-    assert strawberry_union.types == (User, Error)
-
-
-def test_unions_inside_list():
```

Comment:
Why was this test removed?

Reply:
Testing lists of unions was moved to the list tests:
https://github.com/strawberry-graphql/strawberry/pull/1063/files/5f33966dc1aa4a2d0b41c1724fd006b9081cc347#diff-7cc80ffd182a1d8dc3ef0e695dd9d4d137b1d45038ce43bc97a9d4627e0f1c68R44-R60

Reply:
👍